manifest.yml

```
@@ -67,6 +67,11 @@ schema:
    - "https://raw.githubusercontent.com/"
 
 versions:
+  "2.37.2":
```

```suggestion
  "2.38.0":
```

This is a feature release, and needs a semi-major bump.